### PR TITLE
feat: mobile screen redesign

### LINKED ISSUE #398: Mobile screen redesign : Partnership
FIgma file : https://www.figma.com/file/fh0OAvj4AS08kHoSxu3DkE/Starknet-Quest?type=design&node-id=2790%3A4852&mode=design&t=JDjNBExRr0sova6t-1

Loom : In progress...

### LINKED ISSUE #399: Mobile screen redesign : Quest page
Figma file : https://www.figma.com/file/fh0OAvj4AS08kHoSxu3DkE/Starknet-Quest?type=design&node-id=2790%3A14409&mode=design&t=JDjNBExRr0sova6t-1

Loom : In progress...

### LINKED ISSUE #400: Mobile screen redesign : Back button
Figma file : https://www.figma.com/file/fh0OAvj4AS08kHoSxu3DkE/Starknet-Quest?type=design&node-id=2790%3A15158&mode=design&t=JDjNBExRr0sova6t-1

Loom : In progress...

### LINKED ISSUE #401: Mobile screen redesign : Menu
Figma file : https://www.figma.com/file/fh0OAvj4AS08kHoSxu3DkE/Starknet-Quest?type=design&node-id=2791%3A15419&mode=design&t=JDjNBExRr0sova6t-1

Loom : In progress...

### LINKED ISSUE #402: Mobile screen redesign : Profil
Figma file : https://www.figma.com/file/fh0OAvj4AS08kHoSxu3DkE/Starknet-Quest?type=design&node-id=2791%3A16436&mode=design&t=JDjNBExRr0sova6t-1

Loom : In progress...

### LINKED ISSUE #403: Mobile screen redesign : Popup
Figma file : https://www.figma.com/file/fh0OAvj4AS08kHoSxu3DkE/Starknet-Quest?type=design&node-id=2793%3A17570&mode=design&t=JDjNBExRr0sova6t-1

Loom : In progress...

### LINKED ISSUE #404: Mobile screen redesign
Partnership : https://github.com/starknet-id/starknet.quest/issues/398

Quest page : https://github.com/starknet-id/starknet.quest/issues/399

Back button : https://github.com/starknet-id/starknet.quest/issues/400

Menu : https://github.com/starknet-id/starknet.quest/issues/401

Profil : https://github.com/starknet-id/starknet.quest/issues/402

Popup : https://github.com/starknet-id/starknet.quest/issues/403

### PR DESCRIPTION
**Pull request by Marchand-Nicolas**
close: #404 
close: #403 
close: #398 
close: #399 
close: #400
close: #401 
close: #402

### CODE REVIEW

**Review by Kevils — CHANGES_REQUESTED** [review]
1 - There is a slight pixel offset when opening the menu between the logo and the close icon compared to the original ones. Can we fix that? It seems to me that this also happens on Starknet ID

2 - Add more margin (container) on this text(Partnership, 16px left and right
Screenshot 2023-12-27 at 11 46 54

**Comment by Marchand-Nicolas** [comment]
> 1 - There is a slight pixel offset when opening the menu between the logo and the close icon compared to the original ones. Can we fix that? It seems to me that this also happens on Starknet ID

I will do PRs to fix that on Starknet ID

**Comment by Marchand-Nicolas** [comment]
https://github.com/starknet-id/starknet.id/pull/50
https://github.com/starknet-id/app.starknet.id/pull/607

**Review by Kevils — APPROVED** [review]
Lgtm

File: components/UI/navbar.tsx [hunk 1]
```diff
@@ -336,11 +339,11 @@ const Navbar: FunctionComponent = () => {
               </div>
             </div>
             <div className="flex flex-col items-center my-4 w-full">
-              <div className="text-background">
+              <div className={styles.connectButtonContainer}>
                 <Button onClick={onTopButtonClick}>{topButtonText()}</Button>
               </div>
               <div className="flex">
-                <div className="rounded-full shadow-gray-400 p-3 modified-cursor-pointer hover:scale-105 ease-in duration-300 mt-2">
+                <div className="rounded-full shadow-gray-400 p-3 modified-cursor-pointer hover:scale-105 ease-in duration-300 mt-6">
```
Comment (ayushtom) [inline components/UI/navbar.tsx]: should we add this styling in a css class and add it here? seems to be repeated in two places. Would be easier to modify both at once next time from one place

File: components/lands/scene.tsx [hunk 2]
```diff
@@ -59,6 +60,7 @@ export const Scene: FunctionComponent<SceneProps> = ({
 
   const bind = useGesture({
     onDrag: ({ first, down, pinching, cancel }) => {
+      if (window.innerWidth < 768) return;
```
Comment (ayushtom) [inline components/lands/scene.tsx]: in css files we consider mobile as 740px and here 768px. They should have the same value for mobile no?

**Comment by ayushtom** [comment]
lgtm

**Review by ayushtom — APPROVED** [review]

**Pull request merged**